pkg/http/resource_handler.go

```diff
@@ -80,3 +80,17 @@ func (h *Handler) GetStatuses(c *gin.Context) {
 
 	c.JSON(http.StatusOK, statuses)
 }
+
+// Delete handle the namespace deletion.
+func (h *Handler) DeleteResource(c *gin.Context) {
+	namespace := c.Params.ByName("namespace")
+	resource := c.Params.ByName("resource")
+
+	//Delete inventory
+	if err := h.api.DeleteResource(namespace, resource); err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
```

Comment:
I think return a http 500 error code is not a good idea. Maybe return a http.StatusBadRequest instead

Reply:
The error could be any of those. If a user ask to delete an unknown resource it should be a http.StatusBadRequest. But if there was an issue with the kubernetes api then i think a http.StatusInternalServerError is more suited.